prow/plugins/trigger/ic.go

```diff
@@ -62,12 +62,12 @@ func handleIC(c client, trustedOrg string, ic github.IssueCommentEvent) error {
 		// Check for the presence of the needs-ok-to-test label and remove it
 		// if a trusted member has commented "/ok-to-test".
 		if okToTest && ic.Issue.HasLabel(needsOkToTest) {
-			orgMember, err := isUserTrusted(c.GitHubClient, commentAuthor, trustedOrg, org)
+			orgMember, err := trustedUser(c.GitHubClient, trigger, commentAuthor, org, repo)
```

Comment:
This is confusing.. it's creating a var called orgMember, trustedUser doesn't *just* check for org membership.